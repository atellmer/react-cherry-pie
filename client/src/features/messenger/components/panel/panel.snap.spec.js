import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmPanelDesktop from './desktop';
import TmPanelPhone from './phone';


const props = {
  dialogs: []
};

test('Jest: TmPanelDesktop (Shallow Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmPanelDesktop {...props}/>);

  expect(tree).toMatchSnapshot();
});

test('Jest: TmPanelPhone (Shallow Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmPanelPhone {...props}/>);

  expect(tree).toMatchSnapshot();
});

