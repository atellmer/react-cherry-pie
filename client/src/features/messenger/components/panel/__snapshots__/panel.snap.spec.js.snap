// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest: TmPanelDesktop (Shallow Snapshot) 1`] = `
<TmPanelDesktop
  dialogs={Array []}
/>
`;

exports[`Jest: TmPanelPhone (Shallow Snapshot) 1`] = `
<TmPanelPhone
  dialogs={Array []}
/>
`;
